service/awsconfig/v10/adapter/vpc.go

```diff
@@ -56,9 +56,10 @@ func VpcCIDR(clients Clients, vpcID string) (string, error) {
 	output, err := clients.EC2.DescribeVpcs(describeVpcInput)
 	if err != nil {
 		return "", microerror.Mask(err)
-	}
-	if len(output.Vpcs) > 1 {
-		return "", microerror.Maskf(tooManyResultsError, "vpcs: %s", vpcID)
+	} else if len(output.Vpcs) == 0 {
+		return "", microerror.Maskf(notFoundError, "vpc: %s", vpcID)
+	} else if len(output.Vpcs) > 1 {
+		return "", microerror.Maskf(tooManyResultsError, "vpc: %s found %d vpcs", vpcID, len(output.Vpcs))
```

Comment:
Fixes a panic I hit earlier when testing locally. Now we handle if the VPC ID in the awsconfig CR doesn't exist.